docs/modules/player-lifecycle/custom-player-creation.md

```diff
@@ -0,0 +1,136 @@
+<%(TOC)%>
+# Customising player creation
+
+<%(Callout message="
```

Comment:
why is this in a callout?

Reply:
I like how this looks on the page + it grabs more attention than normal text

Reply:
I think the callout is perfect for this and is miles better than what we currently do: 

![image](https://user-images.githubusercontent.com/13353733/55500913-3dba6700-5641-11e9-9b3f-e78ebb93ffd9.png)

Reply:
compared to
![image](https://user-images.githubusercontent.com/10051819/55501122-9f7ad100-5641-11e9-886e-e0a396118266.png)

Reply:
hm.... as long as we make sure that there are not too many callouts, cause then the page is just really colorful

Reply:
and we should be consistent throughout all pages :P